packages/admin/package.json

```diff
@@ -21,6 +21,7 @@
         "generate-graphql-types:watch": "$npm_execpath generate-graphql-types --watch",
         "lint": "run-p lint:eslint lint:tsc",
         "lint:eslint": "eslint --max-warnings 0 src/ package.json",
+        "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
```

Comment:
lint:prettier should be used in lint then, see: https://github.com/vivid-planet/comet/blob/main/demo/admin/package.json#L17

Reply:
I adapted it to fit the `packages/admin/admin` in the comet lib, the link you sent referenced the lint in the `demo` not the `packages`

https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/package.json

see [8a0f6fc](https://github.com/vivid-planet/comet-brevo-module/pull/62/commits/8a0f6fc34cdd2881d5ce225c54d70a9e86fbb5c7)